src/layers.json

```diff
@@ -151,6 +151,22 @@
       "python3.6": "arn:aws:lambda:sa-east-1:464622532012:layer:Datadog-Python36:23",
       "python3.7": "arn:aws:lambda:sa-east-1:464622532012:layer:Datadog-Python37:23",
       "python3.8": "arn:aws:lambda:sa-east-1:464622532012:layer:Datadog-Python38:23"
+    },
+    "us-gov-east-1": {
+      "nodejs10.x": "arn:aws-us-gov:lambda:us-gov-east-1:002406178527:layer:Datadog-Node10-x:30",
```

Comment:
Hmm, did we not publish latest versions to GovCloud?

Reply:
Do you mind helping get the recent versions published?

Reply:
Can do 👍